dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/parameters/AbstractParameters.java

```diff
@@ -201,4 +202,27 @@ private void addPropertyToValPool(Property property) {
         varPool.removeIf(p -> p.getProp().equals(property.getProp()));
         varPool.add(property);
     }
+    /**
+     * Convert the body result returned from HTTP to a map
+     * @param result
+     * @return
+     */
+    public static Map<String, String> getHttpMapByString(String result) {
```

Comment:
I think this method ```getHttpMapByString``` should be included in ```HttpTask``` instead of Abstract class.

Reply:
GetHttpMapByString is moved to HttpParameters because it's used in this